rainforest-cli.go

```diff
@@ -635,8 +635,22 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:         "direct-connect",
+			Usage:        "Start a Rainforest Direct Connect session",
+			OnUsageError: onCommandUsageErrorHandler("direct-connect"),
+			Description:  "Starts a Direct Connect Session for the given tunnel-id",
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "tunnel-id",
+					Usage: "tunnel-id",
```

Comment:
```suggestion
					Usage: "tunnel-id",
					Value: "default",
```

Let's you remove  L33-35 in `direct_connect.go`?